ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+If there are significant objections or questions around proposed changes,
+the ITE Sponsor or Reviewer may choose to return the ITE to a "Draft" status
+for more extensive discussion and eventual <<review, review again for acceptance>>.
+
+[[final]]
+==== Finalizing an ITE
+When the implementation is complete and incorporated into the
+appropriate "main" code repository, the ITE sponsor will change
+the ITE's status changed to "Final".
+
+Active:: [[active]]
+Some Informational and Process ITEs may have a status of "Active" instead of "Final"
+These ITEs are ongoing and never meant to be completed per se. E.g. ITE 1 (this ITE).
+
+[[maintenance]]
+==== ITE Maintenance
+
+Even after a ITE reaches "Final" status, it may need to be updated.
+
+In general, Standards track ITEs are not modified after they have
+reached the Final state. Once a Standards ITE has been completed, Jenkins developer
+documentation must become the formal documentation of the expected behavior.
+
+Informational and Process ITEs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the ITE being updated.
+
+Replaced:: [[replaced]]
+Final ITEs may eventually also be "Replaced" - superseded by a different ITE -
+rendering the original obsolete.
+This is intended for Informational ITEs, where version 2 of an API can replace version 1.
+When a ITE is marked as replaced, the <<header-superseded-by, Superseded-by>> header
+must be filled in with a link to the new ITE.
+
+==== Other ITE Outcomes
+
+Not all ITEs will be accepted and finalized.
+
+Rejected:: [[rejected]]
+A ITE <<Reviewer>> may choose to reject a ITE.
+Perhaps after all is said and done it was not a good idea
+or perhaps a competing proposal is a better alternative.
+It is still important to have a record of this fact.
++
+Rejecting a ITE is a very strong statement.
+If the reviewer believes the ITE might eventually be accepted with sufficient modification,
+the reviewer will not reject the ITE.
+If a reviewer is confident ITE will never be accepted,
+they should inform the ITE sponsor as soon as possible to prevent wasted effort.
+On the other hand, even an <<accepted, Accepted>> ITE may ultimately be rejected
+at some point before it reaches "<<final, Final>>" status,
+due to factors not known at the time it was Accepted.
++
+Upon the request of the sponsor, the reviewer may choose to return a
+Rejected ITE to Draft status, but this is at the discretion of the reviewer.
+
+Withdrawn:: [[withdrawn]]
+A ITE <<Sponsor>> may choose to withdraw a ITE.
+Similar to "Rejected", "Withdrawn" means that the ITE sponsor
+themselves has decided that the ITE is actually a bad idea,
+or agrees that a competing proposal is a better alternative.
+
+Deferred:: [[deferred]]
+A ITE can also be assigned a status of "Deferred". The ITE sponsor or an
+editor can assign the ITE this status when no progress is being made
```

Comment:
Do you want to define a timeframe for when this can happen? 6 months? 1 year?

Reply:
I vote for 6 months, but I'd like to have the opinion of other stakeholders on this decision :)